### DISCUSSION TITLE
potential mocking framework

### DISCUSSION THREAD

**Discussion by reymalahay** [bxparks/AUnit#111] [Q&A]
hi @bxparks,

in issue #109, you said
1) The ability to mock out certain Arduino API functions that were not designed to be mocked

will it be possible to use a mocking framework like [arduinofake](https://github.com/FabioBatSilva/ArduinoFake) in conjunction with `aunit`?

please advise.

thanks in advance,
rey malahay

_Originally posted by @bxparks in https://github.com/bxparks/AUnit/discussions/109#discussioncomment-7372366_

**Comment by bxparks** [comment]
I don't know anything about arduinofake. I took a quick scan, and I get the impression that it's similar to EpoxyDuino, with the addition of a mocking framework, but they all depend on PlatformIO? The documentation is not clear.

If you figure it out, maybe you can update this ticket for the benefit of others. I have other priorities right now, so can't spend any time on this.
